### ISSUE TITLE
Docker: Host our service

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#139]
## User story

1. As {Adorsys}
2. I want / need {DIdentity to be hosted on the provided server via Docker}
3. So that {it's easier to test it independently from a local system}

## Acceptance criteria
* DIdentity runs on server provided by Adorsys with the help of Docker

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
